airbyte-cdk/python/unit_tests/destinations/test_destination.py

```diff
@@ -0,0 +1,234 @@
+import argparse
+import io
+import json
+from os import PathLike
+from typing import Any, Mapping, List, Union, Dict, Iterable, Iterator
+from unittest.mock import ANY
+
+import pytest
+
+from airbyte_cdk.destinations import Destination
+from airbyte_cdk.models import AirbyteRecordMessage, AirbyteStateMessage, AirbyteCatalog, AirbyteConnectionStatus, AirbyteMessage, Type, \
+    ConnectorSpecification, Status, ConfiguredAirbyteCatalog, ConfiguredAirbyteStream, AirbyteStream, SyncMode, DestinationSyncMode
+
+
+@pytest.fixture
+def destination(mocker) -> Destination:
```

Comment:
to avoid unintentional usage of global function instead of fixture
```suggestion
@pytest.fixture(name="destination")
def destination_fixture(mocker) -> Destination:
```

Reply:
what about?
```python
class MockedDestination(Destination):
   def __init__(self):
       self.spec = Mock()
       self.check = Mock()
       self.write = Mock()
```
or 
```python
mocker.patch.object(DummpyDestination, 'spec')
mocker.patch.object(DummpyDestination, 'check')
mocker.patch.object(DummpyDestination, 'write')
```

Reply:
What is the upside of doing the second comment (changing how the object is mocked)? less "hacky"?